app/controllers/shared/api.php

```diff
@@ -358,30 +358,33 @@
         if ($useCache) {
             $resource = null;
             $data = $response->getPayload();
+
             if (!empty($data['payload'])) {
                 $pattern = $route->getLabel('cache.resource', null);
                 if (!empty($pattern)) {
                     $resource = $parseLabel($pattern, $responsePayload, $requestParams, $user);
                 }
 
                 $key = md5($request->getURI() . implode('*', $request->getParams()));
-
                 $data = json_encode([
                 'content-type' => $response->getContentType(),
                 'payload' => base64_encode($data['payload']),
                 ]) ;
 
                 $signature = md5($data);
                 $cacheLog  = $dbForProject->getDocument('cache', $key);
+                $now = date('d-m-Y H:i:s', time());
```

Comment:
With the new `datetime` changes we should use the Datetime class to get the current time. 

Reference 
https://github.com/appwrite/appwrite/blob/0.16.x/app/controllers/general.php#L310

Reply:
fixed